tests/integration/wordpress_client.py

```diff
@@ -166,6 +166,14 @@ def list_plugins(self):
         )
         return [p["plugin"].split("/")[0] for p in response.json()]
 
+    def list_comments(self, status='approve', post_id: int = None):
+        """List all comments in the WordPress site"""
```

Comment:
We have a new standard for docstrings, do you think it applies here? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#docstrings